docs/intro.rst

```diff
@@ -65,11 +65,13 @@ Pywbem also provides a command line utility:
   repository (e.g. on a WBEM server). See :term:`DSP0004` for a definition of
   MOF.
 
-The related `pywbemtools project`_ provides this command line utility:
+The related `pywbemtools project`_ provides these command line utilities:
```

Comment:
How about replacing these with 'the following'

Reply:
DONE - changed as suggested.